float64   width
int32     number_of_lanes
int32     oneway
int64     way_id
